hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -216,6 +230,11 @@ <h3 style="margin-top:40px"> NetCDF Header Info</h3>
 {#        }#}
 {#    })#}
 
+    // netcdf opendap button
+    $(document).ready(function(){
+        $("#btn-netcdf-opendap").insertAfter("#btn-download-all");
```

Comment:
Will this cause netcdf-opendap button to appear always after download-all button? Would the opendap button you defined in your template be sufficient to have the button show up when opendap_link context variable is not empty?